### PR TITLE
Release 23.9.0

### PR DESCRIPTION
**Pull request by jezdez**
Xrefs https://github.com/conda/conda/issues/13080

### CODE REVIEW

**Comment by jezdez** [comment]
Let's wait with merging for the conda-libmamba-solver release that should happen as well tomorrow

File: recipe/meta.yaml [hunk 1]
```diff
@@ -49,25 +49,25 @@ requirements:
...
     - pluggy >=1.0.0
     - tqdm >=4
     - boltons >=23.0.0
     - jsonpatch >=1.32
+    - truststore >=0.8.0           # [py>=310]
   run_constrained:
     - conda-build >=3.18.3
     - conda-content-trust >=0.1.1
     - conda-env >=2.6
     - cytoolz >=0.8.1
```
Comment (cbouss) [inline recipe/meta.yaml:69]:
```suggestion
```
To match https://github.com/conda/conda/compare/23.7.4...23.9.0#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aL56

**Review by cbouss — APPROVED** [review]

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Comment by jezdez** [comment]
FWIW https://github.com/conda-forge/conda-libmamba-solver-feedstock/pull/17 has been opened

**Pull request merged**